bin/.travis/push.sh

```diff
@@ -26,38 +25,32 @@ if [ "$1" = "" ]; then
     exit 1
 fi
 
+if [ "$2" = "" ]; then
+    echo "Argument 2 variable VERSION_FORMAT is not set, format: ezsystems/php. Bailing out !"
+    exit 1
+fi
+
 REMOTE_IMAGE="$1"
+VERSION_FORMAT="$2"
+
 PHP_VERSION=`docker -l error run ez_php:latest php -r "echo PHP_MAJOR_VERSION.'.'.PHP_MINOR_VERSION;"`
-NODE_VERSION=`docker -l error run ez_php:latest-node node -e "console.log(process.versions.node)"` 
+NODE_VERSION=`docker -l error run ez_php:latest-dev node -e "console.log(process.versions.node)"` 
 
 docker images
 docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
 
 ## TAGS
 echo "About to tag remote image '${REMOTE_IMAGE}' with php version '${PHP_VERSION}' and Node '${NODE_VERSION}'"
 
-# "7.0"
-docker tag ez_php:latest "${REMOTE_IMAGE}:${PHP_VERSION}"
-docker tag ez_php:latest-node "${REMOTE_IMAGE}:${PHP_VERSION}-node"
-docker tag ez_php:latest-dev "${REMOTE_IMAGE}:${PHP_VERSION}-dev"
-
 # "7.0-v0"
-if [ "$2" != "" ]; then
-    docker tag ez_php:latest "${REMOTE_IMAGE}:${PHP_VERSION}-${2}"
-    docker tag ez_php:latest-node "${REMOTE_IMAGE}:${PHP_VERSION}-${2}-node"
-    docker tag ez_php:latest-dev "${REMOTE_IMAGE}:${PHP_VERSION}-${2}-dev"
-fi
+docker tag ez_php:latest "${REMOTE_IMAGE}:${PHP_VERSION}-${VERSION_FORMAT}"
+docker tag ez_php:latest-dev "${REMOTE_IMAGE}:${PHP_VERSION}-${VERSION_FORMAT}-dev-node${NODE_VERSION}"
 
 # "latest" (optional)
-if [ "$LATEST" = "$PHP_VERSION" ]; then
+if [ "$LATEST_PHP" = "$PHP_VERSION" ] && [ "$LATEST_NODE" = "$NODE_VERSION" ]; then
     docker tag ez_php:latest "${REMOTE_IMAGE}:latest"
```

Comment:
Not sure what's the better aproach here:
1) Tag latest with version format (latest-v2, latest-v2-dev)
2) Remove tagging latest on v1 format

Right now they both formats use the same naming scheme for latest, so they will overwrite each other.

Reply:
Definitely 2

As for this code block, maybe it should be:
```
if [ "$LATEST_PHP" = "$PHP_VERSION" ] && [ "$LATEST_NODE" = "$NODE_VERSION" ]; then
    docker tag ez_php:latest "${REMOTE_IMAGE}:latest"
    docker tag ez_php:latest-node "${REMOTE_IMAGE}:latest-node${NODE_VERSION}"
    if [ "$LATEST_PHP" = "$PHP_VERSION" ] && [ "$LATEST_NODE" = "$NODE_VERSION" ]; then
        docker tag ez_php:latest-node "${REMOTE_IMAGE}:latest-node"
    fi
fi
```

Implying here a few things:
- The intermediate tag name is called dev but should be called node, as dev is now part of both
- The final node tags still contain dev in their name, but there is no need for that anymore

Reply:
👆 updated 😅

Reply:
Yeah, I've realized that too and decided to rename it to latest-node 😄 

> The intermediate tag name is called dev but should be called node, as dev is now part of both

I've decided (after your previous comment) to merge the -dev-interim and `base` together, so that it has xdebug (disabled by default) and other utility packages - the Node flavour adds just node, nothing else.

Reply:
And I like your suggestion about tagging latest-node<version>, added!